bindings/java/examples/src/RequestFundsFromFaucet.java

```diff
@@ -0,0 +1,42 @@
+// Copyright 2022 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+import org.iota.Wallet;
+import org.iota.types.*;
+import org.iota.types.account_methods.SyncAccount;
+import org.iota.types.exceptions.InitializeWalletException;
+import org.iota.types.exceptions.NoFundsReceivedFromFaucetException;
+import org.iota.types.exceptions.WalletException;
+import org.iota.types.ids.account.AccountAlias;
+import org.iota.types.secret.StrongholdSecretManager;
+
+public class RequestFundsFromFaucet {
+    
+    public static void main(String[] args) throws WalletException, InterruptedException, InitializeWalletException, NoFundsReceivedFromFaucetException {
+        // This example assumes that a wallet has already been created using the ´CreateWallet.java´ example.
+        // If you have not run the ´CreateAccount.java´ example yet, run it first to ensure that the wallet can be loaded correctly.
+        Wallet wallet = new Wallet(new WalletConfig()
+                .withClientOptions(new ClientConfig().withNodes(Env.NODE))
+                .withSecretManager(
+                        new StrongholdSecretManager(Env.STRONGHOLD_PASSWORD, null, Env.STRONGHOLD_SNAPSHOT_PATH))
+                .withCoinType(CoinType.Shimmer)
+        );
+
+        // Get account and sync it with the registered node to ensure that its balances are up-to-date.
+        AccountHandle a = wallet.getAccount(new AccountAlias(Env.ACCOUNT_NAME));
+        AccountBalance balance = a.syncAccount(new SyncAccount().withOptions(new SyncOptions()));
```

Comment:
So we sync here twice, and in the requestFundsFromFaucet as well, is all that necesary?

Reply:
only `a.syncAccount()` is synching here, so its not twice
and the inner of requestFundsFromFaucet() needs to sync too, to check if funding worked

Reply:
the sync after is not really needed true (since the inner of requestFundsFromFaucet() is already doing it)
but better illustrate it to the user

Reply:
dont we auto sync with a delay on creating account? :) but yea to illustrate the point is good enough for me